src/components/20-molecules/dropdown/story.js

```diff
@@ -35,6 +36,7 @@ storyDropdown.add('Dropdown', () => {
       name="${name}"
       dataTestId="${dataTestId}"
       error="${error}"
+      maxHeight="${maxHeight}"
```

Comment:
please add `maxHeight` also to the react example

Reply:
thx